open_samus_returns_rando/files/levels/s000_surface.lua

```diff
@@ -69,6 +69,7 @@ function s000_surface.OnShipStartPointTeleport(_ARG_0_, _ARG_1_)
   end
 end
 function s000_surface.InitFromBlackboard()
+  Scenario.InitGUI()
```

Comment:
Why does surface not have this placed under main()?

Reply:
It was how I did it initially but changed it and forgot it here.